### ISSUE TITLE
Scrolling not working in "iPhone Mirroring" in macOS 15 Beta 2 [bug]

### ISSUE THREAD

**Issue by CyberChun0329** [noah-nuebling/Mac-Mouse-Fix#1015]
## Description

Scrolling not working in "iPhone Mirroring" in macOS 15 Beta 2, disabling Mac Mouse Fix solves the issue. Mos (another smooth scrolling app) doesn't have this issue.

## Steps to reproduce

–

## Context

Mac Mouse Fix Version: 2.2.4
macOS Version: 15
Mouse Model: Keychron M6

## Additional Comments

–

---

### Console Logs

–

### Crash Reports

–

**Comment by Mmahaha** [comment]
I have the same issue and I'm looking forward to a solution :)

**Comment by Nivek-Uil** [comment]
I have the same issue too.

**Comment by powerycz** [comment]
Same issues...

**Comment by valloo-daniel** [comment]
same here

**Comment by noah-nuebling** [comment]
Hey guys, thanks for the feedback!

I can't currently debug this, since my iPhone is older and since I'm in the EU, where iPhone mirroring is not available. 

However, I just ordered a new iPhone yesterday, and got a non-EU Apple ID so I can look into providing a fix soon.

Not sure how complicated this will be to fix / how long it will take. But I plan to prioritize shipping an update that makes iPhone mirroring not totally broken with Mac Mouse Fix at least.

**Comment by leonho** [comment]
@noah-nuebling is there a way to disable the Mac Mouse Fix feature for a specific app (i.e. iPhone Mirroring app)? That could be useful in this situation.

**Comment by noah-nuebling** [comment]
Hey @leonho, that's currently not possible. But there's sort of a workaround: 

If you enable the Menu Bar Item, you can use that to quickly switch off Mac Mouse Fix via the menu bar.

I know this is not a great solution. I have plans for implementing something much better, but I can't give a timeline for when that might come out.

Also, on the iPhone mirroring stuff, I'm unfortunately not sure I can address that very soon. I hope to be able to do it before the end of the year, but I cannot promise that.

**Comment by Caffeine19** [comment]
No, it didn't work now. Don't know why it would work several minutes ago.

---

Hi everyone, did you try turning off the trackpad simulation? It would make scrolling much smoother.

<img width="497" height="293" alt="Image" src="https://github.com/user-attachments/assets/f3469e38-58a5-4ad6-8bae-cee7c1c00cb3" />

---

Update again, It works now.

**Comment by zacharykeeping** [comment]
Any update on this? This is basically unusable on an MX Master 3S. I don't have Track Simulation on so it's not that either.
